src/Stratis.Bitcoin.IntegrationTests.Common/EnvironmentMockUpHelpers/NodeBuilder.cs

```diff
@@ -172,6 +172,14 @@ protected string GetNextDataFolderName(string folderName = null)
             return dataFolderName;
         }
 
+        public void StartAll()
```

Comment:
This has been removed I believe.

Reply:
Yes. It got removed after I was already using it. I assume it got removed based on the belief that nobody was using it.

Reply:
I removed it again as I can do without it.